tests/fixtures/finger_unlock_activity.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "e05b1c22-71d1-44b8-8a4c-e729174a8972",
+  "timestamp": 1726770544000,
+  "icon": "https://d33mytkkohwnk6.cloudfront.net/app/ActivityFeedIcons/finger_unlock@3x.png",
+  "action": "finger_unlock",
```

Comment:
Is it possible to lock with finger as well?  

ie is there a `finger_lock`?

Reply:
Great catch! I wasn't expecting there would be, as there isn't a `pin_lock` action defined in the code. However, after testing, I discovered that there are `finger_lock` and `pin_lock` actions. I have added support for the `finger_lock` action in this PR, and I will open another PR to add the `pin_lock` action.